bin/install_pepys.ps1

```diff
@@ -20,82 +20,113 @@ $icon_string = "$icon_path,0"
 #
 # Add Pepys Import shortcut to Send To
 #
+try {
+    $shortcut_loc = $sendto_location + "\Pepys Import.lnk"
+
+    if (Test-Path $shortcut_loc) { Remove-Item $shortcut_loc; }
+
+    $Shortcut = $WshShell.CreateShortcut($shortcut_loc)
+    # We need to use full paths here or the shortcut will assume everything is relative to
+    # C:\
+    $Shortcut.TargetPath = [System.IO.Path]::GetFullPath(".\pepys_import_sendto.bat")
+    $Shortcut.IconLocation = $icon_string
+    # If we don't set the working directory then we won't be able to import other DLLs or use relative paths
+    # to our Python executable
+    $Shortcut.WorkingDirectory = [System.IO.Path]::GetFullPath(".")
+    $Shortcut.Save()
+}
+catch {
```

Comment:
the introduction of `try/catch` is great. I've just done some Googling, and have seen we can access a description of the error:`
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_try_catch_finally?view=powershell-7#accessing-exception-information

Reply:
Ah great. I've added a print out of all the error information each time we catch something - as with Pepys itself, I have it printing out the exception details first, followed by our `ERROR: Couldn't do X` string.